crates/jwk/src/lib.rs

```diff
@@ -418,40 +422,40 @@ impl JWK {
         match (&self.params, &other.params) {
             (
                 Params::RSA(RSAParams {
-                    modulus: Some(n1),
-                    exponent: Some(e1),
-                    ..
-                }),
+                                modulus: Some(n1),
```

Comment:
I think there is a formatting issue here. Did you run `cargo fmt`?

Reply:
@timothee-haudebourg Yikes!  I did.  Should be good now.